supervisely/video_annotation/video_annotation.py

```diff
@@ -627,6 +627,8 @@ def from_json(
 
         if key_id_map is not None:
             key_id_map.add_video(video_key, data.get(VIDEO_ID, None))
+        else:
+            key_id_map = KeyIdMap()
```

Comment:
```
actual_key_id_map = key_id_map

if actual_key_id_map is None:
       actual_key_id_map = KeyIdMap()
       
if actual_key_id_map is not None:
            key_id_map.add_video(video_key, data.get(VIDEO_ID, None))
            
use actual_key_id_map below

video_ann = cls(
            img_size=img_size,
            frames_count=frames_count,
            objects=objects,
            frames=frames,
            tags=tags,
            description=description,
            key=video_key,
        )
if key_id_map is None:
      video_ann.set_temp_ key_id_map(actual_key_id_map)
      # warn access.... temporary key id map by using video_ann.get_temp./..

return video_ann
```

Reply:
```
video_ann = cls(
            img_size=img_size,
            frames_count=frames_count,
            objects=objects,
            frames=frames,
            tags=tags,
            description=description,
            key=video_key,
        )
if key_id_map is None:
      video_ann.set_temp_ key_id_map(temp_key_id_map)
      # logger.debug

return video_ann
```

Reply:
UPD: empty `key_id_map = KeyIdMap()` initialization will break backward compatibility.

- rolled back to the original
- updated docstring